soaks/tests/splunk_hec_route_s3/README.md

```diff
@@ -0,0 +1,10 @@
+# Splunk HEC -> Route Transform -> s3
```

Comment:
Just a note that I noticed we stopped updating the index of soaks in `soaks/README.md`. I think that's fine and would prefer to just see one of these README files for each soak test.

Reply:
Tell you what, I fully forgot about that index. If you wanted to make a PR to remove that I'd be for it. 

I'm also happy to nuke it later.